.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.0.2-rc.0
+current_version = 3.0.0-rc.1
```

Comment:
Why did this reset back to 3.0.0?

Reply:
This was an artifact of testing, I'll fix